.github/workflows/tutorial.yml

```diff
@@ -0,0 +1,91 @@
+name: Tutorials
```

Comment:
I like the idea to have an independent workflow for tutorials but most of the steps duplicate the ci workflow steps. An alternative to split the those workflows could be to define a new GitHub action to setup/execute markdown-runner.

Reply:
I agree with you that duplicating steps is not optimal. I think there's mechanisms we could use to compose different workflows together.

I haven't had time to study these, but there's https://docs.github.com/en/actions/tutorials/create-actions/create-a-composite-action and https://docs.github.com/en/actions/how-tos/reuse-automations/reuse-workflows.

Maybe we could figure out something nice for that?
The question is, would this be a follow-up work to tidy-up the CI or would you prefer to have this in this PR?

Asking the question because it might take some time to converge to a solution, and Gary's work might depend on this PR for his own tutorial.